app/src/main/java/com/sdpteam/connectout/validation/EventCreationValidator.java

```diff
@@ -19,18 +20,11 @@ public static boolean isValidEventDescription(String description) {
     }
 
     public static boolean eventCreationValidation(EditText eventTitleInput, EditText eventDescriptionInput) {
-        if(!isValidEventTitle(eventTitleInput.getText().toString())) {
-            eventTitleInput.setError(EVENT_TITLE_ERROR);
-            eventTitleInput.requestFocus();
-            return false;
-        }
-        if(!isValidEventDescription(eventDescriptionInput.getText().toString())) {
-            eventDescriptionInput.setError(EVENT_DESCRIPTION_ERROR);
-            eventDescriptionInput.requestFocus();
+        if(!handleValidationFailure(isValidEventTitle(eventTitleInput.getText().toString()), eventTitleInput, EVENT_TITLE_ERROR)
+                || !handleValidationFailure(isValidEventDescription(eventDescriptionInput.getText().toString()), eventDescriptionInput, EVENT_DESCRIPTION_ERROR)){
             return false;
         }
         return true;
     }
```

Comment:
I would advise against this approach, the condition in the if statement could be just returned directly:
`return !handleValidationFailure(...) || !handleValidationFailure(...);`

Reply:
I agree here